install/operator/build/conf/config-test.yaml

```diff
@@ -32,7 +32,7 @@ Syndesis:
             Enabled: false
             Tag: "fabric8/s2i-java:3.0-java8"
             SHA: "fabric8/s2i-java:3.0-java8@sha256:4d13fcc6eda389d4d679602171e11593eadae9b9"
-            CamelVersion: "2.23.2.fuse-760024"
+            CamelVersion: "3.1.0"
             CamelKRuntime: "0.3.4.fuse-740008"
```

Comment:
I don't think this version of Camel K supports this version of Camel. We need to change the Camel K runtime to 1.1.0 or newer.

Reply:
I think @zregvart is right and @nicolaferraro can comment on this. But I believe it's as simple as bumping camel-k version.

Reply:
Just updated the camel-k-runtime, thanks for the review.